src/disco/mux/fd_mux.c

```diff
@@ -544,6 +532,20 @@ fd_mux_tile( fd_cnc_t *              cnc,
       continue;
     }
 
+    ulong sig = fd_frag_meta_sse0_sig( seq_sig );
+    if( FD_UNLIKELY( callbacks->before_frag ) ) {
+      int filter = 0;
+      callbacks->before_frag( ctx, (ulong)this_in->idx, seq, sig, &filter );
```

Comment:
Does moving the call to this callback not change it?

Reply:
It had to be moved, because otherwise we don't know seq yet (there may not even be a new message)